styles/find-and-replace.less

```diff
@@ -166,6 +166,25 @@ atom-workspace.find-visible {
       margin-right: @component-padding;
     }
   }
+
+  .wrap-icon {
+    @wrap-size: 96px;
```

Comment:
Maybe `@wrap-size: @font-size * 10;`?

When using the `icon-move-up` and `icon-move-down` (see https://github.com/atom/find-and-replace/pull/572#issuecomment-155346088), they need a bit of top/bottom padding since they take up the full height. Something like this:

```less
@wrap-size: @font-size * 10;
@wrap-icon-size: @wrap-size * 0.66;
```

and then

```less
&:before {
  font-size: @wrap-icon-size;
  line-height: @wrap-size;
  height: @wrap-size;
  width: @wrap-size;
}
```